en/streaming-integrator/docs/develop/installing-siddhi-extensions.md

```diff
@@ -3,203 +3,130 @@
-- **Installation Locations** - After following the instructions, the resultant jar should be placed in its specific location, based on the following table.
-
-    <table>
-        <tr>
-            <th>Location</th>
-            <th>Directory</th>
-            <th>Jar Types</th>
-        </tr>
-        <tr>
-            <td rowspan=2>runtime</td>
-            <td>
-                <code>&lt;SI_HOME&gt;/lib</code> or <code>&lt;SI_HOME&gt;/bundles</code> based on the instructions.
-            </td>
-            <td>
-                <b>bundle in runtime:</b> An OSGi bundle should be placed in this directory. Conversion instructions will be available in the <code>instructions</code>.
-            </td>
-        </tr>
-        <tr>
-            <td>
-                <code>&lt;SI_HOME&gt;/jars</code>
-            </td>
-            <td>
-                <b>jar in runtime:</b> A non-OSGi bundle can be placed in this directory.
-            </td>
-        </tr>
-        <tr>
-            <td rowspan=2>samples</td>
-            <td>
-                <code>&lt;SI_HOME&gt;/samples/sample-clients/lib</code>
-            </td>
-            <td>
-                <b>jar in samples:</b> A non-OSGi bundle can be placed in this directory.
-            </td>
-        </tr>
-        <tr>
-          <td></td>
-          <td>
-            <b>bundle in samples</b> is <b>not</b> applicable.
-          </td>
-        </tr>
-    </table>
-
-Installation of extensions of which, manually installable dependencies are present, will not be complete until you install these dependencies manually.
+The dialog box displays all the dependencies that need to be manually installed. For each dependency, the dialog box provides the following information.
+
+- **Instructions** to download (and depending on the dependency, to convert) the JAR of the dependency.
+
+- **Installation Location** where the downloaded JAR (and depending on the dependency, the converted OSGi bundle) needs to be placed in order to install the dependency. The following table especifies the directory in which you need to place the JAR/OSGi bundle depending on the installation location.
+
+    |**Installation Location**|**Directory**|
+    |---------------------|-----------------------------------------------------------------------------------------------------------------------------------------------|
+    |**bundle in runtime**|Place the OSGi bundle(s) you downloaded/converted in either the `<SI_HOME>/lib` or the `<SI_HOME>/bundles` directory based on the instructions.|
+    |**jar in runtime**   |Place the non-OSGi bundle(s) you downloaded in the `<SI_HOME>/jars` directory.                                                                 |
+    |**jar in samples**   |Place the non-OSGi bundle(s) you dowloaded in the `<SI_HOME>/samples/sample-clients/lib` directory.                                            |
```

Comment:
Please change "bundle(s)" to "bundle" in each Installation Location. Only a single bundle is referred here.

Reply:
Done